_posts/2022-04-06-debezium-1.9-final-released.adoc

```diff
@@ -0,0 +1,178 @@
+which could become a bit cumbersome when dealing with tens or even hundreds of databases.
+
+Over the last year, Sergei Morozov and his team at SugarCRM reworked the SQL Server connector and the Debezium connector framework to be _multi-partition aware_:
+the framework is now capable of streaming changes from multiple _source partitions_,
+which are split up between _connector tasks_ (in Kafka Connect terminology),
+which in turn can be distributed amongst the worker nodes of a Kafka Connect cluster.
+
+In case of the SQL Server connector, a logical database equates to one such source partition,
+so that you now can stream for instance 20 databases from one physical SQL Server host,
+spread across four source tasks running on five nodes of a Connect cluster.
+To use the new multi-partition mode, configure the names of the databases to capture using the new https://debezium.io/documentation/reference/stable/connectors/sqlserver.html#sqlserver-property-database-names[`database.names`] connector configuration property (rather than using the previously existing `database.dbname`),
+and optionally set the value of `tasks.max` to a value larger than 1.
+Note that the schema and topic names as well as the structure of connector metrics differs between single and multi-partition mode,
+so as to account for the name of the logical database and the id of the source task, respectively.
+
+[.centered-image.responsive-image]
+====
+++++
+<img src="/assets/images/multi_partition_metrics.png" style="max-width:90%;" class="responsive-image">
+++++
+====
+
+Multi-partition mode is experimental as of the 1.9 release and is planned to fully replace the legacy single partition mode for the SQL Server connector in a future release,
+i.e. also if you'd capture changes from only one single logical database,
+you'll be using the multi-partition mode then.
+Multi-partition mode will also be rolled out for other connectors where it's possible, e.g. for the connectors for Oracle and IBM Db2.
+
+Thanks a lot to Sergei and team for their excellent collaboration around that feature!
+
+== Further Changes
+
+Let's take a look at some more features new in Debezium 1.9.
+First, Debezium Server has now a sink adaptor for HTTP,
+which means it can be used as a "native" event source for Knative Serving,
+without the need for sending messages through a message broker like Apache Kafka first.
+
+Then, the friendly folks over at Redis Labs stepped up and contributed several improvements to how Debezium (Server) integrates with Redis Streams: besides several performance improvements,
+the database history for connectors like the MySQL one can now be stored in Redis,
+also offsets which can be stored there now.
+But they didn't stop there: for instance, Debezium Server now supports custom configuration providers,
+as already known in Kafka Connect.
+
+Going forward, the Redis Labs teams is planning on further cool improvements to Debezium at large,
```

Comment:
```suggestion
Going forward, the Redis Labs teams are planning on further cool improvements to Debezium at large,
```
or
```suggestion
Going forward, the Redis Labs team is planning on further cool improvements to Debezium at large,
```
I'm not sure if "team" should be plural or not, but that influences the verb form that follows.

Reply:
Yeah, "team" is what I meant to say. Thx!